package.json

```diff
@@ -181,9 +181,9 @@
   "dependencies": {
     "debug": "~4.1.1",
     "execa": "~3.2.0",
+    "got": "~11.6.0",
     "hh-mm-ss": "~1.2.0",
     "mkdirp": "~0.5.1",
-    "request": "~2.88.0",
     "streamify": "~0.2.9",
```

Comment:
@BarryCarlyon maybe this dependency is no more necessary?

got has built-in stream support

Reply:
I missed that when I was having a mare with git commit-ing

Reply:
You on about line 187?

I thought you were saying I had lleft a request dep in?

Reply:
I'll take a look see and see what else needs to be changed when I get a second